### DISCUSSION TITLE
你好，配置文件有吗，比如我想要代码区域和html区域，上下显示。

### DISCUSSION THREAD

**Discussion by liangyuchao123** [vuejs/repl#318] [Q&A]
我希望html区域在上，代码区域在下

**Comment by moonsky-all** [comment]
没找见成型的文档，但是源码里有，可以试试

![image](https://github.com/user-attachments/assets/687e38c1-9920-48af-9456-abcaec2cbcf9)

**Comment by liangyuchao123** [comment]
感谢，我已经找到了，我还发现可以翻转

**Comment by liangyuchao123** [comment]
但是现在，我想把repl引入自己的项目里面，遇到了许多问题，你有使用经验吗

**Comment by moonsky-all** [comment]
用的不多，这个`@vue/repl`一个页面好像同时只能存在一个实例，有多个实例的话，就只有一个有效，其他的就没效果了

**Comment by liangyuchao123** [comment]
我测试了一下，是可以同时存在多个的
